easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
```

Comment:
@boegel I also prefer to build from sources when possible. I have written a easyconfig which builds from sources

https://github.com/hpcugent/easybuild-easyconfigs/pull/3783